DEPS

```diff
@@ -4,7 +4,13 @@ vars = {
   'github': 'https://github.com',
 
   'effcee_revision': '35912e1b7778ec2ddcff7e7188177761539e59e0',
-  'googletest_revision': 'd9bb8412d60b993365abb53f00b6dad9b2c01b62',
+
+  # For now, pin an older googletest revision
+  'googletest_revision': '1fb1bb23bb8418dc73a5a9a82bbed31dc610fec7',
```

Comment:
It would probably be useful to document what/why this is and point to the documentation at this revision, so tests don't grow dependencies on new `googletest` versions.

Reply:
I filed https://github.com/KhronosGroup/SPIRV-Tools/issues/5039 to follow up